assets/src/components/dummyDetourPage.tsx

```diff
@@ -18,16 +18,11 @@ export const DummyDetourPage = () => {
 
   return (
     <>
-      {routePattern && routePattern.shape && (
+      {route && routePattern && routePattern.shape && (
         <DiversionPage
           originalRoute={{
-            shape: routePattern.shape,
-            routeName: routePattern.routeId,
-            routeDescription: routePattern.headsign || "?",
-            routeOrigin: routePattern.name,
-            routeDirection:
-              route?.directionNames[routePattern.directionId] || "?",
```

Comment:
Lol I forgot that I put that in there as `"?"` - I would have been interested to see the bug report when a dispatcher saw `Inbound`/`Outbound`/`?`